### ISSUE TITLE
freight does not generate Contents File

### ISSUE THREAD

**Issue by Yrrsinn** [freight-team/freight#33]
For main component of wheezy distribution for amd64 architecture, this file would normally be located at

`/dists/wheezy/main/Contents-amd64.gz`

["Contents" indices](https://wiki.debian.org/RepositoryFormat#A.22Contents.22_indices) files are needed for example by the [apt-file tool](https://wiki.debian.org/apt-file), it would be really nice to be able to use this tool with an repository created by freight.

**Comment by mattock** [comment]
:+1: for the idea. How would we create such content indices automatically? Can you provide a PR?

**Comment by Yrrsinn** [comment]
Yes, this issues serves more or less as a reminder for me (I need the feature). Unfortunately I cannot work at once on this.
